extensions-contrib/azure-extensions/src/main/java/org/apache/druid/storage/azure/AzureCloudBlobHolderToCloudObjectLocationConverter.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.storage.azure;
+
+import org.apache.druid.data.input.impl.CloudObjectLocation;
+
+/**
+ * Converts a {@link CloudBlobHolder} object to a {@link CloudObjectLocation} object
+ */
+public class AzureCloudBlobHolderToCloudObjectLocationConverter
+    implements ICloudSpecificObjectToCloudObjectLocationConverter<CloudBlobHolder>
+{
+  @Override
+  public CloudObjectLocation createCloudObjectLocation(CloudBlobHolder cloudBlob)
+  {
+    try {
+      return new CloudObjectLocation(cloudBlob.getContainerName(), cloudBlob.getName());
+    }
+    catch (Exception e) {
+      throw new RuntimeException(e);
```

Comment:
In Druid, I see us throw `RE` which extends from `RuntimeException`s why did you choose to throw that here instead of `RE`? I don't know which one is the right one to throw...